modules/nw-bpfman-infw-configure.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * networking/network_security/bpfman-operator/bpfman-operator-about.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="bpfman-infw-configure_{context}"]
+= Configuring Ingress Node Firewall Operator to use the bpfman Operator
+
+The Ingress Node Firewall employs link:https://www.kernel.org/doc/html/latest/bpf/index.html[BPF] programs to implement some of its key firewall functionality. By default these BPF programs are loaded into the Kernel using a mechanism specific to the Ingress Node Firewall.
+
+As a cluster administrator, you can configure the Ingress Node Firewall Operator to the use bpfman Operator to use link:https://bpfman.io/[bpfman] to load and manage these programs instead, adding additional security and observability functionality.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have an account with administrator privileges.
+* You installed the Ingress Node Firewall Operator.
+
+.Procedure
+
+* Edit the `IngressNodeFirewallConfig` object named `ingressnodefirewallconfig` and set `ebpfProgramManagerMode` field:
```

Comment:
@davegord  I think within OCP, bpfman is named "eBPF Program Manager", or something like that? Should all references to bpfman in OCP docs use that?